src/text.rs

```diff
@@ -176,13 +176,14 @@ mod tests {
             .unwrap()
             .unwrap();
 
-        let actual_len: ORE<8, 16, u32> = serde_json::from_str(&ore_len_str).unwrap();
+        let actual_len: ORE<8, 16> = serde_json::from_str(&ore_len_str).unwrap();
         let lengths = Text::query_length(13, &field()).unwrap();
-        let expected_len = lengths.compatible_value(&actual_len).unwrap();
+        let expected_len = lengths.compatible_member(&actual_len).unwrap();
 
-        assert_eq!(expected_len, &actual_len);
+        assert_eq!(expected_len, actual_len);
     }
 
+    // these currently fail for https://github.com/pgcentralfoundation/pgrx/issues/1860
```

Comment:
More specifically, it was unable to find the postgres type `enquo_set_ore_32_8`. That github issues suggests this is an error for vectors. I tried adding a serch path macro as in that issue but still couldn't get this to pass :/

Reply:
Have addressed this - the type `enquo_set_ore_32_8` was incorrect, should have been `enquo_ore_32_8`. However, the tests are still failing even though the type works. Were these passing before? not too sure how they are supposed to work.

Reply:
You make an excellent point.  I have no idea what I was thinking, because the current `main` is broken for me, too, and I can't see how it could ever have worked.  It looks like you've fixed up those problems in your PR already, though, which I apologise you had to do in addition to the upgrade work.

That being said, the test suite *does* pass on `main^` for me, which used the `enquo_set_*` types.  I'm pretty sure the type of that variable should be changed to `enquo_kith_ore_32_8`, rather than `enquo_ore_32_8`, since that's what I renamed the `enquo_set_ore_32_8` type to in the most recent commit.  I did document [what the dickens a Kith even is](https://github.com/enquo/enquo-core/blob/main/rust/src/datatype/kith.rs), if you're curious, although I added those docs after the last release of `enquo-core`, so it doesn't show up on docs.rs.  Perhaps I should cut a new release of that...

Reply:
No problem! Hehe, I like the name `Kith`. Thinks this is all fixed in latest commit - all made a bit more sense when I looked at the pg operator macro functions in the `ore.rs` module!